Error while checking build/O2/o2-cs8 for 56d6bca93844bca98b9b01c3a0db7f890eabe732 at 2022-09-20 08:02:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 450
 93/106 Test #107: test_Framework_test_SingleDataSource ....................***Failed    2.95 sec
[ERROR] pid 26715 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] SEVERE: Device internal-dpl-injected-dummy-sink (26715) returned with 1
99% tests passed, 1 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9855/56d6bca93844bca98b9b01c3a0db7f890eabe732/build_O2_o2-cs8/pretty.html).

--- a/EventVisualisation/Base/include/EventVisualisationBase/ConfigurationManager.h
+++ b/EventVisualisation/Base/include/EventVisualisationBase/ConfigurationManager.h
@@ -24,7 +24,7 @@ namespace o2
 namespace event_visualisation
 {
 /// Version of the software
-const static std::string o2_eve_version = "1.38";
+const static std::string o2_eve_version = "1.40";
 
 /// Configuration Manager allows an easy access to the config file.
 ///

--- a/EventVisualisation/View/src/Screenshot.cxx
+++ b/EventVisualisation/View/src/Screenshot.cxx
@@ -159,39 +159,23 @@ std::string Screenshot::perform(std::string fileName, o2::detectors::DetID::mask
   const auto annotationStateBottom = MultiView::getInstance()->getAnnotationBottom()->GetState();
   MultiView::getInstance()->getAnnotationTop()->SetState(TGLOverlayElement::kInvisible);
   MultiView::getInstance()->getAnnotationBottom()->SetState(TGLOverlayElement::kInvisible);
-
-  TImage* view3dImage = MultiView::getInstance()->getView(MultiView::EViews::View3d)->GetGLViewer()->GetPictureUsingBB();
-
+  TImage* view3dImage = MultiView::getInstance()->getView(MultiView::EViews::View3d)->GetGLViewer()->GetPictureUsingFBO(width * 0.65, height * 0.95);
   MultiView::getInstance()->getAnnotationTop()->SetState(annotationStateTop);
   MultiView::getInstance()->getAnnotationBottom()->SetState(annotationStateBottom);
-
-  scaledImage = ScaleImage((TASImage*)view3dImage, width * 0.65, height * 0.95, backgroundColorHex);
+  CopyImage(image, (TASImage*)view3dImage, width * 0.015, height * 0.025, 0, 0, view3dImage->GetWidth(), view3dImage->GetHeight());
   delete view3dImage;
-  if (scaledImage) {
-    CopyImage(image, scaledImage, width * 0.015, height * 0.025, 0, 0, scaledImage->GetWidth(), scaledImage->GetHeight());
-    delete scaledImage;
-  }
 
-  TImage* viewRphiImage = MultiView::getInstance()->getView(MultiView::EViews::ViewRphi)->GetGLViewer()->GetPictureUsingBB();
-  scaledImage = ScaleImage((TASImage*)viewRphiImage, width * 0.3, height * 0.45, backgroundColorHex);
+  TImage* viewRphiImage = MultiView::getInstance()->getView(MultiView::EViews::ViewRphi)->GetGLViewer()->GetPictureUsingFBO(width * 0.3, height * 0.45);
+  CopyImage(image, (TASImage*)viewRphiImage, width * 0.68, height * 0.025, 0, 0, viewRphiImage->GetWidth(), viewRphiImage->GetHeight());
   delete viewRphiImage;
-  if (scaledImage) {
-    CopyImage(image, scaledImage, width * 0.68, height * 0.025, 0, 0, scaledImage->GetWidth(), scaledImage->GetHeight());
-    delete scaledImage;
-  }
 
-  TImage* viewZYImage = MultiView::getInstance()->getView(MultiView::EViews::ViewZY)->GetGLViewer()->GetPictureUsingBB();
-  scaledImage = ScaleImage((TASImage*)viewZYImage, width * 0.3, height * 0.45, backgroundColorHex);
+  TImage* viewZYImage = MultiView::getInstance()->getView(MultiView::EViews::ViewZY)->GetGLViewer()->GetPictureUsingFBO(width * 0.3, height * 0.45);
+  CopyImage(image, (TASImage*)viewZYImage, width * 0.68, height * 0.525, 0, 0, viewZYImage->GetWidth(), viewZYImage->GetHeight());
   delete viewZYImage;
-  if (scaledImage) {
-    CopyImage(image, scaledImage, width * 0.68, height * 0.525, 0, 0, scaledImage->GetWidth(), scaledImage->GetHeight());
-    delete scaledImage;
-  }
 
   TASImage* aliceLogo = new TASImage(settings.GetValue("screenshot.logo.alice", "alice-white.png"));
   if (aliceLogo->IsValid()) {
     double ratio = (double)(aliceLogo->GetWidth()) / (double)(aliceLogo->GetHeight());
-    ;
     aliceLogo->Scale(0.08 * width, 0.08 * width / ratio);
     image->Merge(aliceLogo, "alphablend", 0.01 * width, 0.01 * width);
   }

--- a/EventVisualisation/View/src/Screenshot.cxx
+++ b/EventVisualisation/View/src/Screenshot.cxx
@@ -210,7 +210,7 @@ std::string Screenshot::perform(std::string fileName, o2::detectors::DetID::mask
   int textLineHeight = 0.015 * height;
 
   for (int i = 0; i < 4; i++) {
-    lines.push_back("");        // guard for empty collision time
+    lines.push_back(""); // guard for empty collision time
     image->DrawText(textX, textY + i * textLineHeight, lines[i].c_str(), fontSize, "#BBBBBB", "FreeSansBold.otf");
   }
   image->EndPaint();

--- a/EventVisualisation/View/src/Screenshot.cxx
+++ b/EventVisualisation/View/src/Screenshot.cxx
@@ -210,6 +210,7 @@ std::string Screenshot::perform(std::string fileName, o2::detectors::DetID::mask
   int textLineHeight = 0.015 * height;
 
   for (int i = 0; i < 4; i++) {
+    lines.push_back("");        // guard for empty collision time
     image->DrawText(textX, textY + i * textLineHeight, lines[i].c_str(), fontSize, "#BBBBBB", "FreeSansBold.otf");
   }
   image->EndPaint();

